src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -466,13 +466,16 @@ def _filter_privacy_request_queryset(
         query = query.filter(PrivacyRequest.id.in_(identities_query))
 
     if custom_privacy_request_fields:
+        # Note that if Custom Privacy Request Field values were arrays, they are not
+        # indexed for searching and not discoverable here
         custom_field_conditions = [
             (CustomPrivacyRequestField.field_name == field_name)
             & (
                 CustomPrivacyRequestField.hashed_value
                 == CustomPrivacyRequestField.hash_value(value)
             )
             for field_name, value in custom_privacy_request_fields.items()
+            if not isinstance(value, list)
```

Comment:
If the field upon which we're searching is a list field I am ignoring it all together, to avoid hashing the list to begin with, and the fact that it won't turn up anything.

Reply:
In other words, it will be as if this query param doesn't exist -